str0m/src/media/mline.rs

```diff
@@ -460,10 +489,76 @@ impl MLine {
 
         Some(NextPacket {
             pt,
-            pkt,
+            seq_no,
             ssrc: pkt.meta.ssrc,
+            body: NextPacketBody::Regular { pkt },
+        })
+    }
+
+    fn create_padding_packet(&mut self, target_size: DataSize, now: Instant) -> Option<NextPacket> {
+        let pt_to_use = {
+            let mut vec: Vec<_> = self
+                .buffers_tx
+                .iter()
+                .map(|(pt, buffer)| (pt, buffer.history_size()))
+                .filter(|(_, size)| *size > 0)
+                .collect();
+            vec.sort_by_key(|(_, size)| *size);
+
+            // Use the last PT i.e. the one that has the most RTX history, this is a poor
+            // approximation for most recent sends.
+            vec.last().map(|(pt, _)| **pt)
+        };
```

Comment:
This looks like we can use `max_by_key` to  simplify.

Reply:
I'll check